pydantic/types.py

```diff
@@ -626,3 +634,238 @@ def display(self) -> str:
 
     def get_secret_value(self) -> bytes:
         return self._secret_value
+
+
+class Color:
+    __slots__ = '_original', '_color_match'
+
+    def __init__(self, value: ColorType) -> None:
+        self._original: ColorType = value
+        self._color_match: Optional[str] = None
+        self._parse_color()
+
+    @staticmethod
+    def _rgb_str_to_tuple(value: str) -> Optional[RGBAType]:
+        """
+        Return RGB/RGBA tuple from the passed string.
+
+        If RGBA cannot be matched, return None
+        """
+        r = re.compile(
+            r'rgba?\((?P<red>\d{1,3}),\s*'
+            r'(?P<green>\d{1,3}),\s*'
+            r'(?P<blue>\d{1,3})'
+            r'(,\s*(?P<alpha>\d{1}\.\d{1,}))?\)',
```

Comment:
```suggestion
            r'(,\s*(?P<alpha>\d{1}\.\d+))?\)',
```

tiny tweak, i think `+` is the same as `{1,}`